dotnet-desktop-guide/net/winforms/wfdev-diagnostics/wfdev005.md

```diff
@@ -0,0 +1,47 @@
+---
+title: WFDEV005 warning
+description: Learn about Windows Forms compile-time warning WFDEV005. 'Clipboard.GetData(string)' is obsolete. Use 'Clipboard.TryGetData<T>' methods instead.
+ms.date: 03/04/2025
+f1_keywords:
+  - "WFDEV005"
+helpviewer_keywords:
+  - "WFDEV005"
+---
+# Compiler Warning WFDEV005
+
+**Version introduced:** .NET 10 Preview 1
+
+> `Clipboard.GetData(string)` is obsolete. Use `Clipboard.TryGetData<T>` methods instead.
```

Comment:
We have 2 more messages associated with this error - 
`DataObject.GetData` methods are obsolete. Use the corresponding `DataObject.TryGetData<T>` instead. 

`ClipboardProxy.GetData(As String)` method is obsolete. Use `ClipboardProxy.TryGetData(Of T)(As String, As T)` instead. 

With 2 different suggersted resolutions